pkg/kubectl/cmd/exec.go

```diff
@@ -48,6 +48,10 @@ var (
 		kubectl exec 123456-7890 -c ruby-container -i -t -- bash -il`)
 )
 
+const (
+	execUsageStr = "expected 'exec POD_NAME COMMAND [ARG1] [ARG2] ... [ARGN]'.\nPOD_NAME and COMMAND are required arguments for the exec command"
```

Comment:
Is this correct? `exec` takes `--` to start the `COMMAND` section. We should make it consistent with the usage, like 

```expected 'exec POD -- COMMAND [args...]'.\nPOD and COMMAND are required ...``` 

or even use the actual usage

```
execUsageStr = "expected %q.\nPOD and COMMAND are required arguments for the exec command."

(...)

return cmdutil.UsageError(cmd, execUsageStr, cmd.Use)
```